test/integrations/routes/profile.test.js

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable no-unused-expressions */
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import jwt from 'jsonwebtoken';
+import app from '../../../index';
+import models from '../../../models';
+
+chai.use(chaiHttp);
+
+const profile = {
+  firstname: faker.name.firstName(),
+  lastname: faker.name.lastName(),
+  phone: faker.phone.phoneNumber(),
+  bio: faker.random.words(),
+  address: faker.address.streetAddress(),
+  gender: 'M',
+  user_id: 1,
+  username: faker.random.words(),
+  image: faker.image.imageUrl(),
+};
+
+const token = `Bearer ${jwt.sign({ user: { id: 1 } }, 'secret', { expiresIn: '24hrs' })}`;
+
+describe('API end point for User profile', () => {
+  after(() => models.Profile.destroy({ truncate: true }));
+  it('it should create profile with correct details', (done) => {
```

Comment:
#### Nitpick

- Your test captions should make a proper English statement,What you have now looks like this `API end point for User profile it should create profile with correct details` .